### ISSUE TITLE
feat(ui): add keyboard based map navigation

### ISSUE THREAD

**Issue by alyec** [fgpv-vpgf/fgpv-vpgf#52]
Keyboard driven pan and zoom should be enabled for accessibility.  Allowing users to tab through to the navigation panel does not count.  Need to wait until some tab order issues are sorted out.

**Comment by AleksueiR** [comment]
Some related notes:
- map should be be the first focusable element in the container
- basemap selector is placed before the basemap trigger in the tab order (should be after)
- legend entry flags cannot be accessed via the keyboard; they should be focusable as well
- show settings toggle when legend entry gains keyboard focus
- keyboard only mode for identify
  - this should be a separate task

**Comment by milesap** [comment]
Should we have similar/identical functionality as found in google maps?

- Arrow keypress pans a set amount
- Arrow keydown continually pans until released
- + - keys for zoom
- On keyup select new location for point information (alternatively make user hit enter)
